flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphConstructionTest.java

```diff
@@ -450,8 +414,8 @@ private void verifyTestGraph(ExecutionGraph eg, JobID jobId,
 		}
 	}
 	
-	@Test
-	public void testCannotConnectMissingId() {
+	@Test(expected = JobException.class)
+	public void testCannotConnectMissingId() throws JobException {
```

Comment:
Do we have to add a `throws JobException` if you've specified a `@Test(expected = JobException.calss)`?